environments/openshift/manifests/telemeter-template.yaml

```diff
@@ -172,6 +172,54 @@ objects:
           - --whitelist={__name__="cluster:network_attachment_definition_instances:max"}
           - --whitelist={__name__="cluster:network_attachment_definition_enabled_instance_up:max"}
           - --whitelist={__name__="insightsclient_request_send_total"}
+          - --whitelist={__name__="cam_app_workload_migrations"}
+          - --whitelist={__name__=~"cluster:usage:.*"}
+          - --whitelist={__name__="up"}
+          - --whitelist={__name__="cluster_version"}
+          - --whitelist={__name__="cluster_version_available_updates"}
+          - --whitelist={__name__="cluster_operator_up"}
+          - --whitelist={__name__="cluster_operator_conditions"}
+          - --whitelist={__name__="cluster_version_payload"}
+          - --whitelist={__name__="cluster_installer"}
+          - --whitelist={__name__="cluster_infrastructure_provider"}
+          - --whitelist={__name__="cluster_feature_set"}
+          - --whitelist={__name__="node_uname_info"}
```

Comment:
Hmm something wrong here, you already have this metric on line 138 of this fine.

Reply:
We need https://github.com/openshift/telemeter/pull/306 to merge first and then bump the telemeter jsonnet dependency. Then the duplicate whitelist entries should disappear.